### DISCUSSION TITLE
change Limit of maximum amount of holidays

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/kimai#4999] [Plugins]
When booking a longer holiday, I get the message "You can book a maximum of 20 days...". Where / how can I change this limit?

**Comment by kevinpapst** [comment]
You can't. Book two vacations.

**Comment by pkoevesdi** [comment]
Thanks. I'm aware of that workaround, but it's inconvenient, and I don't see the reason for it. Can I put it as a feature request to let the admin raise and even switch off this limit?

**Reply by kevinpapst** [reply]
Sure, we can discuss everything. But not here, as this is for the core software.
Please post it here: https://github.com/kimai/Paid-plugins

**Comment by pkoevesdi** [comment]
Ok, sorry, wasn't aware of that. Posted it: https://github.com/kimai/Paid-plugins/issues/181
